services/transactionpool/test/add_transaction_test.go

```diff
@@ -156,8 +157,20 @@ func TestReturnsReceiptForTransactionThatHasAlreadyBeenCommitted(t *testing.T) {
 	})
 }
 
-func TestDoesNotAddTransactionIfPoolIsFull(t *testing.T) {
+func TestDoesNotAddTransactionWithFutureProtocolVersion(t *testing.T) {
```

Comment:
do we need one for `minimal version - 1` also?

Reply:
Regarding transactions protocol check. The use of same protocol versions for transaction and blocks should be reevaluated. 

The definition is transaction.protocol version is represents the protocol version that based on it the transaction was formed. The current assumption is that all lower transactions protocol versions are supported. A breaking change in transaction format or meaning will require a new configuration for minimal supported transaction protocol.